### ISSUE TITLE
Tao Tray Icon Disk Usage

### ISSUE THREAD

**Issue by bagnaram** [yuezk/GlobalProtect-openconnect#398]
**Describe the bug**
The disk usage in the /run/user/1000/tao directory grows indefinitely.
I notice that new tray icons get created every time the VPN connects or disconnects. This causes the directory to grow without bounds filling up the logical volume.

**Expected behavior**
Temp directories should re-use existing resources and clear when closing the program

**Screenshots**
```
ls -al /run/user/1000/tao
.rw-r--r--@  96k user 23 Jul 18:27 tray-icon-04b5b80a-c88a-454b-8e45-dfe9d23bd63b.png
.rw-r--r--@ 104k user 23 Jul 18:27 tray-icon-067e6ff3-8bbe-4511-afec-81a2c06ab939.png
.rw-r--r--@  96k user 23 Jul 18:29 tray-icon-1af6fca5-51f4-4bff-b071-77cd91c81f1b.png
.rw-r--r--@  96k user 23 Jul 18:29 tray-icon-1bf6e108-3dd3-42ea-a780-83f78d43f29c.png
.rw-r--r--@  95k user 23 Jul 18:29 tray-icon-2c62fae1-963c-49c3-8acc-7590f48e9731.png
.rw-r--r--@ 104k user 23 Jul 18:29 tray-icon-4b0c7c7c-a2c0-4f99-a24d-7d95629da0c3.png
.rw-r--r--@  96k user 23 Jul 18:27 tray-icon-5df44748-b494-4c02-97b8-5fffe46ecded.png
.rw-r--r--@ 104k user 23 Jul 18:27 tray-icon-5f4462e0-e2b5-4826-b1d8-0a6422956ce1.png
.rw-r--r--@ 104k user 23 Jul 18:27 tray-icon-7ac809a3-59f3-489e-af91-c0001bdfef0f.png
.rw-r--r--@ 104k user 23 Jul 18:29 tray-icon-23eb0035-9b71-49a8-8b99-2dbf908268bf.png
.rw-r--r--@ 100k user 23 Jul 18:27 tray-icon-45f1bf05-1199-46ef-999c-643a46d3c303.png
.rw-r--r--@ 100k user 23 Jul 18:29 tray-icon-69eba25d-2835-4cf6-949d-bc493720e7d4.png
.rw-r--r--@ 104k user 23 Jul 18:29 tray-icon-71d6b31c-cd31-490f-b0ab-5508279b8731.png
.rw-r--r--@  96k user 23 Jul 18:29 tray-icon-71d3922e-c6e8-4d4f-8ac1-1bd35d54f8ad.png
.rw-r--r--@ 104k user 23 Jul 18:29 tray-icon-301faa2e-8543-4831-bd92-cede2ee09343.png
.rw-r--r--@ 104k user 23 Jul 18:27 tray-icon-739a5eed-8622-4bb5-b78e-e05f0e213e9d.png
.rw-r--r--@  96k user 23 Jul 18:29 tray-icon-3469b974-7e84-4947-a717-b15bca661a0d.png
.rw-r--r--@ 104k user 23 Jul 18:29 tray-icon-443028d1-b829-403f-89f9-44478de56329.png
.rw-r--r--@  96k user 23 Jul 18:29 tray-icon-734487b9-e2d6-4316-9fa2-b1deec24ce4c.png
.rw-r--r--@ 104k user 23 Jul 18:27 tray-icon-2164873f-6957-49d7-aff5-b978b068614e.png
.rw-r--r--@ 104k user 23 Jul 18:29 tray-icon-3769758e-f66d-4f68-a83e-9f07a02f2545.png
.rw-r--r--@ 104k user 23 Jul 18:29 tray-icon-b33f1259-e5d3-453b-bcc0-0461bf00cfec.png
.rw-r--r--@  96k user 23 Jul 18:29 tray-icon-b467e248-1b72-48b3-9e40-79eff2bb9061.png
.rw-r--r--@  96k user 23 Jul 18:27 tray-icon-c4a6570e-0d48-448f-a937-de5860c5b809.png
.rw-r--r--@ 104k user 23 Jul 18:29 tray-icon-e9578490-4f23-4312-9f1d-1b588229f5c7.png
.rw-r--r--@ 104k user 23 Jul 18:29 tray-icon-eeda7cde-7de7-49e6-b3b2-6b5f714f8669.png
.rw-r--r--@  96k user 23 Jul 18:27 tray-icon-f633c2c7-566a-4c46-9d8b-e0dbea7141da.png
.rw-r--r--@  95k user 23 Jul 18:29 tray-icon-fe3b41a6-6de4-48b0-9aa4-e25016f0014b.png
```

**Logs**
- For the GUI version, you can find the logs at `~/.local/share/gpclient/gpclient.log`
- For the CLI version, copy the output of the `gpclient` command.

**Environment:**
 - OS: fedora 6.9.9-200.fc40.x86_64
 - Desktop Environment: Sway

 - Is remote SSH? N

**Additional context**

**Comment by yuezk** [comment]
@bagnaram Thanks for your feedback! Thanks looks like a bug and I will try to fix it.

**Comment by yuezk** [comment]
This has been fixed in 2.3.7. Please upgrade it.
